### PR TITLE
feat(project): add special construction project info text

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/templates/project/form.hbs [hunk 1]
```diff
@@ -80,6 +80,23 @@
         }}
       />
 
+      <div uk-alert class="uk-flex uk-flex-middle uk-form-label">
+        <span>
+          <div class="uk-text-bold">
+            {{t "ember-gwr.specialConstructionWarning.header"}}
+          </div>
+          <div>
+            {{t "ember-gwr.specialConstructionWarning.1"}}
```
Comment (luytena) [inline addon/templates/project/form.hbs:89]: The .1 and .2 seem kinda weird, but I can't think of another way either.

Comment (luytena) [reply addon/templates/project/form.hbs:89]: Maybe part1 and part2 ? I was expecting an option like with the other code labels.

Comment (velrest) [reply addon/templates/project/form.hbs]: I changed it to `part1` and `part2`.

**Review by luytena — APPROVED** [review]

**Pull request merged**